### DISCUSSION TITLE
Error while loading shared libraries: libdmrconf.so.0.9: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by clewisit** [hmatuschek/qdmr#175] [Q&A] [answered]
I just installed this on my computer.  I followed the installation instructions for install from source.  When starting qdmr, I get the message:
qdmr: error while loading shared libraries: libdmrconf.so.0.9: cannot open shared object file: No such file or directory

I did not get any errors during the compile and install process.  I did confirm that the file does exist, it is installed in /usr/local/lib, and symlinked correctly to libdmrconf.so.0.9.1

Computer is AMD Phenom 64 4 core  12GB ram.  Ubuntu 20.04, everything is up to date.

**Comment by hmatuschek** [comment, marked as answer]
The path is correct. Home-brewed software should be installed under `/usr/local/`. 

However, this path is frequently not searched for libraries by default. To change that, add the line `/usr/local/lib` to `/etc/ld.so.conf` or create a new file in `/etc/ld.so.conf.d`containing that line. Then run `sudo ldconfig` to update the linker. 

This should do the trick.

**Comment by Ignisleo** [comment]
Thanks. Had the same problem on a Raspberry Pi 4, 4GB running Raspbian (buster), and it did the trick. Should maybe be a hint at the "install from source" part on the web site?

**Comment by clewisit** [comment]
Thanks.  That worked.  Odd, because there are other files in /etc/ld.so.conf.d that also have that path, but I added the qdmr.conf file and ran sudo ldconfig, and qdmr starts now.

**Comment by hmatuschek** [comment]
> Thanks. That worked. Odd, because there are other files in /etc/ld.so.conf.d that also have that path, but I added the qdmr.conf file and ran sudo ldconfig, and qdmr starts now.

Then, there was only a call to `ldconfig` missing.

**Comment by hmatuschek** [comment]
> Thanks. Had the same problem on a Raspberry Pi 4, 4GB running Raspbian (buster), and it did the trick. Should maybe be a hint at the "install from source" part on the web site?

Updated the install page accordingly.

**Comment by PeterSprague** [comment]
For my Fedora 40 install built from source: /usr/local/lib64

**Reply by n7ekb** [reply]
Same here!
